docs/getting-started/supported-languages-and-tools.md

```diff
@@ -30,7 +29,7 @@ The table below lists all programming languages that Codacy supports and the cor
   </colgroup>
   <thead>
     <tr>
-      <th>Language</th>
+      <th>Language or framework</th>
```

Comment:
Can this introduced confusion with frameworks like Rails, Symphony, Wordpress that are supported by some tools but will not be listed here?

Reply:
Here I was purposefully using "Language or framework" as loose terms that would encompass the new file formats analyzed by Checkov and the existing Visualforce framework, as well as markup languages such as Markdown and XML, and everything else that's not a "proper" programming language like CSS and SQL. :slightly_smiling_face: 

The frameworks that you mention could be considered to be "one layer below" the information presented in this table, although it would be interesting to "unfold" this information and also list the extra frameworks that the tools support directly on this page at some point. :thinking: 

I think the main issue ends up being this: most customers won't know that the tools in this list further support other frameworks not mentioned explicitly here. So although perhaps this won't be confusing as of now, we're not providing all the information that could be useful yet.